packages/editor-ui/src/stores/posthog.ts

```diff
@@ -37,6 +50,24 @@ export const usePostHog = defineStore('posthog', () => {
 		return getVariant(experiment) === variant;
 	};
 
+	// for testing
+	window.featureFlags = {
+		// since features are evaluated serverside, regular posthog mechanism to override clientside does not work
+		override: (name: string, value: string | boolean) => {
+			overrides.value[name] = value;
+			featureFlags.value = {
+				...featureFlags.value,
+				[name]: value,
+			};
+			try {
+				localStorage.setItem(LOCAL_STORAGE_EXPERIMENT_OVERRIDES, JSON.stringify(overrides.value));
+			} catch (e) {}
+		},
+
+		getVariant,
+		getAll: () => featureFlags.value || {},
+	};
```

Comment:
We should move this outside of `usePostHog()`, otherwise the `featureFlags` will be set every time we call the function. Instead, you can add `const { featureFlags } = usePostHog();` inside of the `override` function.

Reply:
wrapped it in an if statement